zonal_datacube/fishnet.py

```diff
@@ -0,0 +1,42 @@
+import geopandas as gpd
+import shapely.geometry as geometry
+import shapely.wkt as wkt
+
+
+def create_fishnet_grid(min_x, min_y, max_x, max_y, cell_size) -> gpd.GeoDataFrame:
+    x, y = (min_x, min_y)
+    geom_array = []
+
+    # Polygon Size
+    while y < max_y:
+        while x < max_x:
```

Comment:
`cell_size` is constant. So would using a `range` make the loop arithmetic go away?